addons/web/static/tests/widgets/data_export_tests.js

```diff
@@ -149,16 +201,72 @@ QUnit.module('widgets', {
             viewOptions: {
                 hasActionMenus: true,
             },
-            mockRPC: this.mockDataExportRPCs,
-            session: {
-                ...this.mockSession,
-                get_file: function (params) {
-                    const data = JSON.parse(params.data.data);
+            mockRPC: function (route, args) {
+                if (route === '/web/dataset/call_kw/ir.async/search_read') {
+                    return Promise.resolve([]);
+                }
+                if (route === '/longpolling/poll') {
+                    self.pollPromise = testUtils.makeTestPromise();
+                    self.pollPromise.abort = () => {self.pollPromise.resolve([], $.Event())};
+                    return self.pollPromise;
+                }
```

Comment:
Same remark with `async mockRPC`. In fact most comments in tests apply to all tests, so to reduce amount of comments I comment only once but should make changes in whole diff in test file.